src/commands/plugins.rs

```diff
@@ -467,6 +467,17 @@ async fn try_install(
     if continue_to_install(manifest, package, yes_to_all)? {
         let installed = manager.install(manifest, package).await?;
         println!("Plugin '{installed}' was installed successfully!");
+
+        if let Some(description) = manifest.description() {
+            println!("\nDescription:");
```

Comment:
Is there a reason we want this additional newline added to a `println` statement? I'm assuming it is a stylistic choice

Reply:
Without some additional line breaks it disappears with the rest of the output. A mild improvement though. I believe #1481 will address this without needing the line breaks.